services/proxy/pkg/config/config.go

```diff
@@ -126,6 +127,11 @@ type JWKS struct {
 	RefreshUnknownKID bool   `yaml:"refresh_unknown_kid" env:"PROXY_OIDC_JWKS_REFRESH_UNKNOWN_KID" desc:"If set to 'true', the JWKS refresh request will occur every time an unknown KEY ID (KID) is seen. Always set a 'refresh_limit' when enabling this." introductionVersion:"pre5.0"`
 }
 
+type ClaimsChecker struct {
+	Name   string `yaml:"name" env:"PROXY_OIDC_CLAIMSCHECKER_NAME" desc:"Name of the claims checker to be used. It can be 'Bool'" introductionVersion:"7.3.0"`
```

Comment:
`Name of the claims checker to be used. It can be 'Bool'` --> This does not sound valid to me.

Are we confident that this gets introduced in 7.3.0?
Otherwise we should use the release name and change it before finalization, see [Introduce New Envvars](https://owncloud.dev/services/general-info/envvars/envvar-naming-scopes/#introduce-new-envvars).

Reply:
I dont know how far the implementation already is, but pls add the missing when implemented.
The names are fine.

Please also add a description in the proxy README because this needs some explaining text.
I will add an admin doc issue and reference this PR.